ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/vmtracer-sessions.j2

```diff
@@ -1,18 +1,18 @@
-{% if vmtracer_sessions is defined and vmtracer_sessions is not none %}
+{% if vmtracer_sessions is arista.avd.defined %}
 
 ## VM Tracer Sessions
 
 ### VM Tracer Summary
 
 | Session | URL | Username | Autovlan | Source Interface |
 | ------- | --- | -------- | -------- | ---------------- |
-{%     for session in vmtracer_sessions | arista.avd.natural_sort %}
-{%         set url = vmtracer_sessions[session].url | arista.avd.default('-') %}
-{%         if vmtracer_sessions[session].autovlan_disable is arista.avd.defined and vmtracer_sessions[session].autovlan_disable == true %}
+{%     for session in vmtracer_sessions | arista.avd.convert_dicts('name') | arista.avd.natural_sort('name') %}
+{%         set url = session.url | arista.avd.default('-') %}
+{%         if session.autovlan_disable is arista.avd.defined and session.autovlan_disable == true %}
```

Comment:
This is fine but would be good to change this line to ```if session.autovlan_disable is arista.avd.defined(true)```

Reply:
I think it's acceptable to make these small fixes/optimizations as we review - fixed in: [29b903b](https://github.com/aristanetworks/ansible-avd/pull/1635/commits/29b903b69268dcfb0c874927deb3a4a78f463165)